src/custom/state/claim/hooks/index.ts

```diff
@@ -739,7 +743,7 @@ function fetchClaims(account: string, chainId: number): Promise<UserClaims> {
       })
       .then((address) => fetchClaimsFile(address, chainId))
       .then((result) => {
-        if (result[formatted]) return transformRepoClaimsToUserClaims(result[formatted]) // mod
+        if (result[lowerCasedAddress]) return transformRepoClaimsToUserClaims(result[lowerCasedAddress]) // mod
```

Comment:
okay, this is the part where we need to agree with @alfetopito . 

I think is best to use always. Also, allows us to remove the `startingAddress.toLowerCase()` in 735 (although not a big deal)

Reply:
Did you mean @fedgiac ?

Reply:
Remove the lowercasing from all addresses